net/mdns-repeater/src/etc/rc.syshook.d/carp/50-mdns

```diff
@@ -0,0 +1,45 @@
+#!/usr/local/bin/php
+<?php
+
+require_once("config.inc");
+require_once("util.inc");
+require_once('interfaces.inc');
+require_once("plugins.inc.d/mdnsrepeater.inc");
+
+if (mdnsrepeater_carp_enabled()) {
+    $subsystem = !empty($argv[1]) ? $argv[1] : '';
+    $type = !empty($argv[2]) ? $argv[2] : '';
+
+    if ($type != 'MASTER' && $type != 'BACKUP') {
+        log_error("Carp '$type' event unknown from source '{$subsystem}'");
+        exit(1);
+    }
+
+    if (!strstr($subsystem, '@')) {
+        log_error("Carp '$type' event triggered from wrong source '{$subsystem}'");
+        exit(1);
+    }
+
+    list ($vhid, $iface) = explode('@', $subsystem);
+
+    $friendly_interface = convert_real_interface_to_friendly_interface_name($iface);
+
+    if (!in_array($friendly_interface, mdnsrepeater_interfaces())) {
+        exit(0);
+    }
+
+    switch ($type) {
+        case 'MASTER':
+            syslog(LOG_NOTICE, "Starting mDNS repeater\n");
+            if (!file_exists("/var/run/mdns-repeater.pid")) {
+                shell_exec('/usr/sbin/service mdns-repeater onestart');
```

Comment:
onexxx is for ignoring missing or incomplete configuration. Use plain start and stop please. Should work fine.